scripts/start.sh

```diff
@@ -48,17 +48,18 @@ ISINIT=false
 BUILDAPPLICATION=false
 MAKESAMPLEDATA=false
 SERVERMODE=false
+NOTIMESTAMP=false
 
 USAGE="Usage: explorer-eosio start [-dev] [-d] [-b] [-s] [--init] [--server-mode] (program to start eosio-explorer)
 
 where:
     -dev, --develop     Starts the tool in development mode
     -d, --delete        Removes existing Docker containers
-    -b, --build         Re-build gui
+    -b, --build         Build gui
     -s, --sample-data   Starts the tool with pre-existing sample accounts and smart contracts
-    --init              Builds a production-ready version of the web tool,
-                          and opens the tool with cleared local storage
-    --server-mode       Starts the tool in server-mode, it will start the dockers but not the gui"
+    --init              Opens the tools with cleared local storage
```

Comment:
I think we should rename init to clear-local-storage because init is confusing here.
We already have eosio-explorer init and people may think this does the same. 
But this init is just clearing local storage so using clear-local-storage makes more sense.

Reply:
ok, I will follow your suggestion ( actually I want to do the same though! )

Reply:
done